app-land/scss/component/_button.scss

```diff
@@ -0,0 +1,15 @@
+.cta__button {
+  &--button {
+    width: 7rem;
+    height: 3rem;
+    background: $vibrantGreen;
+    border: none;
+    border-radius: 2rem;
```

Comment:
You can use border radius variables from shared folder when you need border radius in your project  :)